src/SortedArrayView.hpp

```diff
@@ -86,28 +86,38 @@ class SortedArrayView
    * @brief Default move constructor, performs a shallow copy.
    * @param src The SortedArray to be moved from.
    */
-  LVARRAY_HOST_DEVICE inline
+  LVARRAY_HOST_DEVICE constexpr inline
```

Comment:
Is there a reason we would want to have a `LVARRAY_CONSTEXPR`?

Reply:
We don't have an `LVARRAY_CONSTEXPR`, although now we do have an `LVARRAY_INTEL_CONSTEXPR` which is a bad name because it is only `constexpr` if not using the intel compiler.

Reply:
I thought it really only applies to a `constexpr` function as well? Perhaps we should have a `LVARRAY_CONSTEXPR` and an `LVARRAY_CONSTEXPR_FUNC`...didn't we used to have that?

Reply:
I think so but why should we define a macro that just expands to `constexpr`? We wouldn't ever think of doing something like `#define LVARRAY_CONST const` so I don't see why `constexpr` should be any different.